sources/packages/backend/apps/db-migrations/src/migrations/1719602073438-AddVirusScanColumns.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddVirusScanColumns1719602073438 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
It works, but its better to split the migrations with respect to the number of types and tables, as it will be helpful for migration rollback checks and even naming the migration name with the table name or type name.